locale/defaultMessages.json

```diff
@@ -406,6 +406,10 @@
     "context": "section header",
     "string": "Recent Orders"
   },
+  "1MyMcE": {
+    "context": "table header column",
+    "string": "Tax Rate"
```

Comment:
Should we capitalise all words in messages?

Reply:
I confirmed with design team that many capitalisations are unnecessary and can be deleted. Gonna fix it